content/concepts/module-resolution.md

```diff
@@ -3,70 +3,73 @@ title: Module Resolution
 sort: 8
 contributors:
     - pksjce
+    - pastelsky
 ---
 
-A resolver is a library which helps find the absolute path of a module.
-A module can be required as a dependency from another module as
+A resolver is a library which helps in locating a module by its absolute path.
+A module can be required as a dependency from another module as:
 
 ```js
-import mymodule from 'path/to/module'
+import module from 'path/to/module'
```

Comment:
Hmm, `module` is already kind of a reserved variable (`module.exports`). It won't give errors to use `module` (AFAIK), but it can be very confusing. Perhaps rename to `foo` or something?